specification/connector/_types/Connector.ts

```diff
@@ -254,6 +254,7 @@ export interface Connector {
   api_key_secret_id?: string
   configuration: ConnectorConfiguration
   custom_scheduling: ConnectorCustomScheduling
+  deleted?: boolean
```

Comment:
Reading https://github.com/elastic/elasticsearch/pull/118669/ it looks like `deleted` is always set.

```suggestion
  deleted: boolean
```